indico/modules/events/abstracts/templates/management/abstract_list_filter.html

```diff
@@ -31,6 +32,19 @@
     </div>
 {% endmacro %}
 
+{% macro _render_boolean_filter(id=none, name=none, label=none) %}
+    <div id="form-group-tracks_required" class="form-group">
+        {% if label %}
+            <div class="form-label">
+                <label for="{{ id }}">{{ label }}</label>
+            </div>
+        {% endif %}
+        <div class="form-field">
+            {{ switch(id=id, name=name, checked=filters['extra'][id] | bool) }}
```

Comment:
i think without spaces around the `|` it's more readable here (or `()` around the whole `filters['extra'][id] | bool` but i'd go for removing these two spaces instead)